sematic/docker_images.py

```diff
@@ -0,0 +1,46 @@
+# Standard Library
```

Comment:
I usually try to say "container image" instead of "docker image" to remain generic and non-branded, so we could name this file `container_images.py`, but that's just a nit.

Would also make it consistent with the env var name.

Reply:
done